### DISCUSSION TITLE
[403 Error] Forbidden on eNotices2

### DISCUSSION THREAD

**Discussion by SkySoul95** [OP-TED/eForms-SDK#323] [Q&A] [answered]
Hello and sorry for my bad english.
I'm facing this issue with eNotices2 Preview: I have created preview API Key and I created and account on the eNotices2 Website but It seems that my key doesn't let me access to any service. May you help me to understand why? I get the same error from a self-developed software. I have read the previous answers on this topic but nothing helped me

![image](https://user-images.githubusercontent.com/5845129/216580480-9369edec-d10c-43fa-858d-5859f0819548.png)
 
Thank you in advance,
Have a nice day!

**Comment by pcolast** [comment, marked as answer]
Hello,

You have created your API key in Preview, but it seems you didn't create your eNotices2 account in Preview.
Could you please log into https://enotices2.preview.ted.europa.eu/home (with the same EU login account that you used to generate your API key) and then retry the API calls ?

Have a nice day,
Stefan

**Reply by SkySoul95** [reply]
Thank you! I solved it two days ago when I have created an account on preview environment too (I only had an account in production). Now it answers correctly, thank you again.

Have a nice day,
Matteo
